content/collections/experiment-apis/en/experiment-evaluation-api.md

```diff
@@ -5,42 +5,49 @@ title: 'Experiment Evaluation API'
 source: 'https://www.docs.developers.amplitude.com/experiment/apis/evaluation-api/'
 summary:  Lets you retrieve variant assignment data for users with remote evaluation.
 ---
-The Amplitude Experiment Evaluation REST API lets you retrieve variant assignment data for users via [remote evaluation](/docs/feature-experiment/remote-evaluation). User information passes as query parameters on the request to allow for [caching the response on the CDN](/docs/feature-experiment/under-the-hood/performance-and-caching#cdn-caching). When you call this API, Amplitude tracks an `[Experiment] Assignment` event.
+
+Amplitude Experiment's evaluation APIs lets your re
+
+The Amplitude Experiment Evaluation APIs lets you retrieve variant assignment data for users via [remote evaluation](/docs/feature-experiment/remote-evaluation) using the evaluation API, or download local evaluation. User information passes as query parameters on the request to allow for [caching the response on the CDN](/docs/feature-experiment/under-the-hood/performance-and-caching#cdn-caching). When you call this API, Amplitude tracks an `[Experiment] Assignment` event.
```

Comment:
```suggestion
The Amplitude Experiment Evaluation APIs retrieve variant assignment data for users through [remote evaluation](/docs/feature-experiment/remote-evaluation) with the evaluation API, or download local evaluation. User information passes as query parameters on the request to allow for [caching the response on the CDN](/docs/feature-experiment/under-the-hood/performance-and-caching#cdn-caching). When you call this API, Amplitude tracks an `[Experiment] Assignment` event.
```

what does `download local evaluation` mean here?

Instead of retrieving remote evaluation from Amplitude servers, you can download the variant assignments that happened locally?